subprojects/gtk4-0.9-rs.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = gtk4-0.9.0
+source_url = https://crates.io/api/v1/crates/gtk4/0.9.0/download
```

Comment:
Why 0.9.0 and not 0.9.5 btw?

Reply:
Not sure... anyway Cargo.lock uses 0.9.2, so I updated both it and meson to 0.9.5